holidayextras/ui-brand-chessington
# Owl Carousel

#### What does this PR do? (please provide any background)

Adding in owl carousel to chessington brand

#### What tests does this PR have?

No tests 

#### How can this be tested?

Checkout this branch inside pattern library
View organisms/owl carousel
You should see the carousel


#### What gif best describes how you feel about this work?
![](http://media3.giphy.com/media/R7HJd0w8dbZzq/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
